### DISCUSSION TITLE
Inactive Timers?

### DISCUSSION THREAD

**Discussion by tango2590** [babybuddy/babybuddy#583] [General]
What is the reason behind retaining "inactive timers? I understand it's needed for unassigned tasks, but if a timer is stopped and attached to an activity (feeding, sleep, etc), is there a need to retain them? Once an activity is attached to a timer, it should be cleared. In my mind, keeping completed timers could result in duplicate activity entries .

**Comment by cdubz** [comment]
See #109. The default behavior is changed as part of ongoing work for a 2.x branch.
